StepsToCare/CareWorkers.ascx

```diff
@@ -59,9 +59,22 @@
 
                 <div class="row">
                     <div class="col-md-4">
-                        <Rock:LocationPicker ID="lpGeofenceLocation" runat="server" Label="Geofence Location" AllowedPickerModes="Named" Help="If geofence is set it is used for auto assignment of care needs based on where the person's home is located. Only named Polygon/Geofence locations are supported to minimize where you can create the area for use in multiple entities such as Groups."/>
+                        <Rock:LocationPicker ID="lpGeofenceLocation" runat="server" Label="Geofence Location" AllowedPickerModes="Named" Help="If geofence is set it is used for auto assignment of care needs based on where the person's home is located. Only named Polygon/Geofence locations are supported to minimize where you can create the area for use in multiple entities such as Groups." />
                     </div>
-                    <div class="col-md-2">
+                    <div class="col-md-4">
+                        <Rock:NumberRangeEditor ID="nreAgeRange" runat="server" Label="Age Range" Help="If using 'Include Family' checkbox on Care Entry it will take into account age ranges for family members" />
```

Comment:
```suggestion
                        <Rock:NumberRangeEditor ID="nreAgeRange" runat="server" Label="Age Range" Help="If using 'Include Family' checkbox on Care Entry it will take into account age ranges for appropriate family members" />
```